DEV-9/DEV-9/PageObjects/MailRu/MailInboxPage.cs

```diff
@@ -0,0 +1,63 @@
+﻿using OpenQA.Selenium;
+
+namespace DEV_9.PageObjects.MailRu
+{
+    /// <summary>
+    /// Inbox page.
+    /// </summary>
+    public class MailInboxPage
+    {
+        /// <summary>
+        /// The WebDriver.
+        /// </summary>
+        private IWebDriver driver;
+
+        /// <summary>
+        /// The button for writing a new letter.
+        /// </summary>
+        public IWebElement WriteNewLetterButton => this.driver.FindElement(By.XPath("//a[@data-name='compose']"));
+
+        /// <summary>
+        /// The latest mail in the inbox.
+        /// </summary>
+        public IWebElement LatestMail => this.driver.FindElement(By.XPath("//div[@class='b-datalist__body']/div[1]"));
```

Comment:
not a good way of finding elements '[1]'.

Maybe it is better to store collection of the elements and only then get first from it

findElementS